actors/builtin/reward/reward_logic.go

```diff
@@ -5,8 +5,30 @@ import (
 	big "github.com/filecoin-project/specs-actors/actors/abi/big"
 )
 
-var BaselinePowerAt = func(epoch abi.ChainEpoch) abi.StoragePower {
-	return big.NewInt(1 << 40) // PARAM_FINISH
+const BaselineExponentString = "340282663082994238536867392845056089438"
+
+// Baseline function = BaselineInitialValue * (BaselineExponent) ^(t), t in epochs
+var BaselineExponent big.Int     // Q.128
```

Comment:
@Kubuxu please check my `Q.x` correctness.  This is my first time doing precision tracking this way.

Reply:
Seems alright.